keras_hub/src/models/retinanet/retinanet_backbone.py

```diff
@@ -0,0 +1,84 @@
+import keras
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.feature_pyramid_backbone import FeaturePyramidBackbone
+from keras_hub.src.models.retinanet.feature_pyramid import FeaturePyramid
+from keras_hub.src.utils.keras_utils import standardize_data_format
+
+
+@keras_hub_export("keras_hub.models.RetinaNetBackbone")
+class RetinaNetBackbone(FeaturePyramidBackbone):
+    def __init__(
+        self,
+        backbone,
```

Comment:
to follow the pattern used in other models, if the `backbone` here is used for feature extraction, let us call this `image_encoder`

Reply:
Done